src/Facing.php

```diff
@@ -56,6 +56,9 @@ private function __construct(){
 		self::EAST
 	];
 
+	/**
+	 * @deprecated
+	 */
 	public const OFFSET = [
```

Comment:
why is this marked as deprecated?

Reply:
It's marked as obsolete because i'm replacing it with a function. In my opinion, it is clearer.

With constant:
```
Facing::OFFSET[Facing::DOWN->value]
```

With function:
```
Facing::DOWN->offset()
```

Reply:
This seems outside of the PR scope